docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -658,7 +658,41 @@ REGEXP = ^\s*((math(\s+|$)|inline(\s+|$)|display(\s+|$)))+
  - `ALLOW_ATTR`: The attribute this policy allows. Must be non-empty.
  - `REGEXP`: A regex to match the contents of the attribute against. Must be present but may be empty for unconditional whitelisting of this attribute.
 
-You may redefine `ELEMENT`, `ALLOW_ATTR`, and `REGEXP` multiple times; each time all three are defined is a single policy entry.
+**Note**: The above section naming policy is new; previously the section was `[markup.sanitizer]` and keys could be redefined.
+Now, a unique identifier must appear in the section name (e.g., `[markup.sanitizer.TeX]`) in order to parse multiple rules.
+This was changed because the implementation with the ini parser used was flawed; the following configs were indistinguishable after parsing:
+
+```ini
+[markup.sanitizer]
+ELEMENT = a
+ALLOW_ATTR = target
+REGEXP = $1
+ELEMENT = a
+ALLOW_ATTR = rel
+REGEXP = $2
+ELEMENT = img
+ALLOW_ATTR = src
+REGEXP = $3
+```
+
+and
+
+```ini
+[markup.sanitizer]
+ELEMENT = a
+ALLOW_ATTR = target
+REGEXP = $1
+ELEMENT = img
+ALLOW_ATTR = rel
+REGEXP = $2
+ELEMENT = img
+ALLOW_ATTR = src
+REGEXP = $3
+```
+
+Because of limitations in the ini library, we are unable to automatically migrate configurations.
+
+We will still parse the first rule from a `[markup.sanitizer]` section if present, but multiple rules must be manually migrated.
```

Comment:
```suggestion
```
I'd rather remove this section. If you feel like an explanation is needed, please reference the version where it was introduced and link the relevant issue or this PR (e.g. "this was changed from the original implementation in 1.11 due to severe limitations; see `#1234`"). But we tend to avoid making explicit version references in the docs, so maybe just remove it and let the Blog speak. 😁

Reply:
If a blog entry is fine with @zeripath I could write such, but I must confess I've never looked at the blog entries and mostly read documentation. :-) [On the breaking](https://github.com/go-gitea/gitea/releases/tag/v1.11.0) section of release notes, there's a link to the PR, but not to the blog entry -- do none of those have blog entries?

Reply:
Hmm... So this bit could be dropped and instead placed in the PR description - during release we can then precis this and put it into the blog post.